extensionsGallery-insider.json

```diff
@@ -1023,6 +1023,63 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "29",
+					"extensionName": "sqldm-performance-insights",
+					"displayName": "Idera SQL DM Performance Insights",
+					"shortDescription": "SQL Diagnostic Manager performance insights dashboard",
+					"publisher": {
+						"displayName": "Idera Inc",
+						"publisherId": "IDERA",
+						"publisherName": "IDERA"
+					},
+					"versions": [
+						{
+							"version": "0.4.0",
+							"lastUpdated": "12/3/2018",  
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [  
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",
+									"source": "https://www.idera.com/productssolutions/freetools/sqldmperformanceinsights"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+									"source": "https://www.idera.com/AppVisor/icon.png"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Content.Details",
+									"source": "https://www.idera.com/AppVisor/README.md"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Code.Manifest",
+									"source": "https://www.idera.com/AppVisor/package.json"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Content.License",
+									"source": "https://www.idera.com/AppVisor/license.rtf"
+								}
+							],
+							"properties": [
+								{
+									"key": "Microsoft.VisualStudio.Code.ExtensionDependencies",
+									"value": ""
+								},
+								{
+									"key": "Microsoft.VisualStudio.Code.Engine",
+									"value": *"
+								},
+								{
+									"key": "Microsoft.VisualStudio.Services.Links.Source",
+									"value": "https://www.idera.com/"
+								}
+							]
+						}
+					],
+					"statistics": [],
+					"flags": "preview"
 				}
 			],
 			"resultMetadata": [
```

Comment:
please bump up the count number as you have added a new one in this PR.

Reply:
count bumped to 19